src/apiCalls/apiCalls.js

```diff
@@ -0,0 +1,12 @@
+export const getMoviesData = () => {
+  return fetch('https://swapi.co/api/films/')
+    .then(response => {
+      if (!response.ok) {
```

Comment:
Ah! Forgot to do this, i remember I had to do it from Overlook last mod.  Thanks for adding!